### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by jerivas** [stephenmcd/mezzanine#2000] [General]
This is the first post on the new GitHub Discussions platform for Mezzanine. Hello to all fellow Mezzanine users! 👋

**Comment by molokov** [comment]
Hello! Thank you for continuing to maintain Mezzanine. It's still one of the best Django-based CMS platforms, and long may it continue to be so!

**Comment by rapto** [comment]
Yes, another thank you from Spain. Please let me know if there is something I can help in, I have lots of experience, but not so much time.

**Comment by QbasicFan** [comment]
Thx, for letting us know. Mezzanine is a great tool.

**Comment by skru** [comment]
Good idea!

**Comment by csotelo** [comment]
Long time I have no see changed on the project, any suggest?

**Reply by kenbolton** [reply]
The issue tracker has been quite active lately.

What kinds of change are you looking for?

**Reply by csotelo** [reply]
I am not looking for a change or issue, I just was looking for information about the project. I have been using it some time ago and I was thinking on retake it, but I have noticed that tere were no changes on two years, then I was thinking on how could I help to this project that, on my point of view is a great project

**Reply by molokov** [reply]
The good news is that I recently updated to Django 5.0 and no extra changes were needed to Mezzanine!
(I have raised a few issues recently too, and attempted to fix them, although I think my pull requests have extra commits in them, due to my fork - couldn't work out how to resolve this, sorry!)

**Reply by csotelo** [reply]
Nice

I hope I could help on it :) Bests
